python/ecl/__init__.py

```diff
@@ -80,15 +81,27 @@
 #    module should contain the variable lib_path pointing to the
 #    directory with shared object files.
 try:
-    import __ecl_lib_info
-    ecl_lib_path = __ecl_lib_info.lib_path
-    ert_so_version = __ecl_lib_info.so_version
-    __version__ = __ecl_lib_info.__version__
+    from .__ecl_lib_info import EclLibInfo
+    ecl_lib_path = EclLibInfo.lib_path
+    ert_so_version = EclLibInfo.so_version
+    __version__ = EclLibInfo.__version__
 except ImportError:
     pass
 except AttributeError:
     pass
 
+#This are python2 -> python3 intermediate arithmetic functions:
+
+#Same functionality as for python2 return x/y: 
+#if either x or y is a float the return value is a float
+#otherwise the result i rounded down (5/2 -> 2, -5/2 -> -3)
+def python2_div(x, y):
```

Comment:
This does not seem like a good solution. I would rather import the new division for python2 and change the implementation to use `//` where its needed, if it is *really* needed.

Reply:
I agree with Jens, `from future import division` together with `//` where needed should be enough.